irispy/sji.py

```diff
@@ -0,0 +1,158 @@
+        Temporal index to stop MapCube at
+
+    skip: `int`
+        Temporal index to skip over MapCube at
+
+
+    Returns
+    -------
+    iris_cube: `sunpy.map.MapCube`
+        A map cube of the SJI sequence
+    """
+
+    if type(filelist) == str:
+        filelist = [filelist]
+    iris_cube = sunpy.map.MapCube()
+    for fname in filelist[0:]:
+        newmap = sunpy.instr.iris.SJI_to_cube(fname)
+        for frame in newmap[start:stop:skip]:
+            if frame.mean() < frame.max():
+                cmap = cm.get_cmap(frame._get_cmap_name())
+                vmax = frame.mean()+3.*frame.std()
+                frame.plot_settings['cmap'] = cmap
+                frame.plot_settings['norm'] = colors.LogNorm(1, vmax)
+                #  todo: iris_intscale
+                iris_cube.maps.append(frame)
+
+    #  todo: pointing correction(rot_hpc)
+
+    #  Option to overlay sun coordinates
+    if grid:
+        iris_cube[0].draw_grid(linestyle='--', color='orange')
+
+    return iris_cube
+
+
+def save_to_mp4(ani, outputfile, fps=60):
+    '''
+    Set up ffmpeg writer
+    Save an animation of the MapCube
+    Example:
+    mc = sunpy.map.Map(files, cube=True)
+    ani = mc.plot()
+    save_to_mp4(ani, "myfile.mp4")
+
+    Parameters
+    ----------
+    ani:`matplotlib.animation.FuncAnimation`
+        Input animation
+    Outputfile: `str`
+        Path name of output file (.mp4)
+    fps: `int`
+        Desired frames per seconds (default=60)
+
+    Return
+    -----------
+    NoneType
+        mp4 to outputfile path
+    '''
+
+    plt.rcParams['animation.ffmpeg_path'] = '/usr/local/bin/ffmpeg'
+    writer = animation.FFMpegWriter(fps=fps, metadata=dict(artist='SunPy'),
+                                    bitrate=64000)
+    if not writer.isAvailable():
+        path = input('Set FFMpeg path: (ex. /usr/local/bin/ffmpeg)')
+        plt.rcParams['animation.ffmpeg_path'] = path
+    writer = animation.FFMpegWriter(fps=fps, metadata=dict(artist='SunPy'),
+                                    bitrate=64000)
+    if not writer.isAvailable():
+        print('Cannot find available FFMpeg Writer')
+        return
+
+    ani.save(outputfile, writer=writer)
+    print('Successfully saved to ' + outputfile)
+
+
+def dustbuster(mc):
+    """
+    Read SJI fits files and return Inpaint-corrected fits files.
+    Image inpainting involves filling in part of an image or video
+    using information from the surrounding area.
+
+    Parameters
+    ----------
+    mc: `sunpy.map.MapCube`
+        Mapcube to read
+
+
+    Returns
+    -------
+    mc: `sunpy.map.MapCube`
+        Inpaint-corrected Mapcube
+    """
+    image_result = []
+    ndx = len(mc)
+    for i, map in enumerate(mc):
+        image_orig = map.data
+        nx = map.meta.get('NRASTERP')
+        firstpos = range(ndx)[0::nx]
+        #  Create mask with values < 1, excluding frame (-200)
+        mask = np.zeros(image_orig.shape)
+        mask[np.where(image_orig < 1)] = 1
```

Comment:
I changed it to use numpy mask array, which returns a boolean array. It works, however I get this warning  when trying to inpaint the new values:

VisibleDeprecationWarning: using a boolean instead of an integer will result in an error in the future
Should I use  np.where(m.mask) here? 
        for layer in range(image_orig.shape[-1]):
            image_orig[m.mask] = image_inpaint[m.mask]